### PR TITLE
[V11][V10] l10n_in_hr_payroll advice creation issue & CLA

### PR DESCRIPTION
**Pull request by tpa-odoo**
Fix for bug: https://github.com/odoo/odoo/issues/26690

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_in_hr_payroll/models/l10n_in_hr_payroll.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def cancel_sheet(self):
 
     @api.onchange('company_id')
     def _onchange_company_id(self):
-        self.bank_id = self.company_id.partner_id.bank_ids and self.company_id.partner_id.bank_ids[0].bank_id.id or False
+        self.bank_id = self.company_id.partner_id.bank_ids and self.company_id.partner_id.bank_ids[0].bank_id and self.company_id.partner_id.bank_ids[0].bank_id.id or False
```
Comment (pedrobaeza) [inline addons/l10n_in_hr_payroll/models/l10n_in_hr_payroll.py]: Put directly `.id` and remove the new `and...` and the `or False` part.

Comment (mreficent) [reply addons/l10n_in_hr_payroll/models/l10n_in_hr_payroll.py]: In brief, revert the change

File: addons/l10n_in_hr_payroll/models/l10n_in_hr_payroll.py [hunk 2]
```diff
@@ -136,7 +136,7 @@ def create_advice(self):
                         'company_id': company.id,
                         'name': run.name,
                         'date': run.date_end,
-                        'bank_id': company.partner_id.bank_ids and company.partner_id.bank_ids[0].id or False
+                        'bank_id': company.partner_id.bank_ids and company.partner_id.bank_ids[0].bank_id and self.company_id.partner_id.bank_ids[0].bank_id.id or False
```
Comment (mreficent) [inline addons/l10n_in_hr_payroll/models/l10n_in_hr_payroll.py]: also here revert change. you only need to put `.bank_id.id` instead of `.id`

**Comment by mart-e** [comment]
Hi,

Please rename your cla file to end with a `.md` to be considered by the CLA bot.

**Comment by tpa-odoo** [comment]
Done

**Comment by mart-e** [comment]
Thanks, merged in 11.0

abb2059ed95 [FIX] l10n_in_hr_payroll: use correct parameter
7383ca18c06 [CLA] signature for tpa-odoo

**Pull request closed without merging**